src/Shared/UnitTests/Vacancies.Client/Infrastructure/Extensions/VacancyExtensionsTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using AutoFixture;
+using Esfa.Recruit.Vacancies.Client.Application.Providers;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Extensions;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.Vacancy;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.ApprenticeshipProgrammes;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace Esfa.Recruit.Vacancies.Client.UnitTests.Vacancies.Client.Infrastructure.Extensions
+{
+    public class VacancyExtensionsTests
+    {
+        private const string _documentId = "doc_id";
```

Comment:
Because this is a const could it be called `DocumentId`

Reply:
👍